core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala

```diff
@@ -75,16 +76,18 @@ private[spark] class HeartbeatReceiver(sc: SparkContext, clock: Clock)
   // "spark.network.timeout" uses "seconds", while `spark.storage.blockManagerSlaveTimeoutMs` uses
   // "milliseconds"
   private val slaveTimeoutMs =
-    sc.conf.getTimeAsMs("spark.storage.blockManagerSlaveTimeoutMs", "120s")
+    sc.conf.getTimeAsMs("spark.storage.blockManagerSlaveTimeoutMs",
```

Comment:
The val `slaveTimeoutMs` is only used in https://github.com/apache/spark/blob/c6ff59a230758b409fa9cc548b7d283eeb7ebe5d/core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala#L80, so I don't think this change shall fix anything.

Reply:
cc @zsxwing to confirm.

Reply:
I look at this carefully, I think your are right, thanks @jiangxb1987 . One case that is not relevant with this PR is like this: set spark.storage.blockManagerSlaveTimeoutMs=900ms and not configure spark.network.timeout, then `executorTimeoutMs ` will be 0 since getTimeAsSeconds loos precision for ms. This config maybe not reasonable. If need fix how about add ensuring > 0 or make executorTimeoutMs's min value as 1, @jiangxb1987 @zsxwing

Reply:
Looks like we never use `slaveTimeoutMs`. Let's delete this `val` and just assign this value to `executorTimeoutMs`.

Reply:
I have removed temp val `slaveTimeout`, also `timeoutIntervalMs` is the same case, so removed too, thanks @zsxwing @jiangxb1987